appstore/api/v1/serializers.py

```diff
@@ -23,6 +23,7 @@ class InstanceSerializer(serializers.Serializer):
     aid = serializers.CharField(allow_null=True)
     sid = serializers.CharField()
     fqsid = serializers.CharField()
+    workspace_name = serializers.CharField()
```

Comment:
Does this need to support `empty` ?

Reply:
Yes, I think it should support empty string. I will go ahead and make that change.